packages/paste-core/components/blockquote/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,102 @@
+import { render, screen } from "@testing-library/react";
+import * as React from "react";
+
+import { Blockquote, BlockquoteContent, BlockquoteSource } from "../src";
+
+describe("Blockquote", () => {
+  it("should render", () => {
+    render(
+      <Blockquote url="#" data-testid="blockquote">
+        <BlockquoteContent>This is some text.</BlockquoteContent>
+        <BlockquoteSource author="Google" source="People + AI Guidebook" />
+      </Blockquote>,
+    );
+
+    const blockquote = screen.getByTestId("blockquote");
+    expect(blockquote).toBeDefined();
+
+    const text = screen.getByText("This is some text.");
+    expect(text.nodeName).toBe("BLOCKQUOTE");
+    expect(blockquote.querySelector(`[data-paste-element='BLOCKQUOTE_SOURCE_ANCHOR']`)).toBeTruthy();
+  });
+
+  it("should render without a url", () => {
+    render(
+      <Blockquote data-testid="blockquote">
+        <BlockquoteContent>This is some text.</BlockquoteContent>
+        <BlockquoteSource author="Google" source="People + AI Guidebook" />
+      </Blockquote>,
+    );
+
+    const blockquote = screen.getByTestId("blockquote");
+    expect(blockquote).toBeDefined();
+    expect(blockquote.querySelector(`[data-paste-element='BLOCKQUOTE_SOURCE_TEXT']`)).toBeTruthy();
```

Comment:
There is no test to be sure `BLOCKQUOTE_SOURCE_ANCHOR` does not exist. Should also be testing for `BLOCKQUOTE_SOURCE_CITE` in these too